docs/reference/contributing/guidelines/workflow.md

```diff
@@ -219,11 +219,29 @@ We use many other labels to summarize the scope and effect of the changes.
+
+When a pull request is selected to be integrated into a rollup pull requests, it already has a release label and is patiently waiting to start CI testing. Once selected to be looped into a rollup pull request, the original pull request will gain a *rollup PR* label. When the rollup pull request is generated, a comment will also be added to the original pull request mentioning that no further work needs to be done to it. 
+
+In the event that the rollup pull request fails CI testing, maintainers will identify the offending pull requests, update their statuses, and will provide additional guidance on what went wrong. Additionally, if a pull request is updated *while* it is already in a rollup pull request, and the rollup pull request passes CI and is merged, the pull request that was updated *will not* be automatically merged.
+
+##### How it works
+
+Rollup pull requests take advantage of the exact same process that is used to merge a pull request into master, only instead using a temporary branch that has been freshly rebased off of master. All pull requests with the *rollup PR* label are locally cloned and merged into this temporary branch, and assuming no merge conflicts arise, a pull request is opened against this temporary branch. Once the rollup pull request is merged, all other pull requests that went into building the rollup pull request are also _marked_ as merged because their contents are already a part of master.
+
+Rollup pull requests are a solution to two types of problem: CI testing duration, and semantic conflict problem. The most obvious use of a rollup pull request is to drastically lower the time needed to test many pull requests at once. Best case, maintainers go from needing to put N pull requests through CI, down to only one, drastically lowering the load on CI infrastructure, and helps close pull requests much sooner. The second, more suble, problem that rollup pull requests solve is the case where two pull requests pass testing on their own, but as soon as they join together, the way they interact with each other causes tests to fail. A simple example can be found [here](https://bors.tech/essay/2017/02/02/pitch).
+
+When merged, rollup pull requests will automatically cause the pull requests that made it to also be marked as merged as well. The commits that made up the original pull requests are also apart of the rollup pull requests, so when the rollup pull request is merged, the changes in the pull requests are already merged and the pull request is updated accordingly. This is also why if a new change is added to a pull request _after_ the rollup pull request is made, it _will not_ automatically be merged once the rollup pull request is, because all changes the updated pull request are still not in master.
```

Comment:
Nitpick
"When merged, rollup pull requests will automatically cause the pull requests that made it to also be marked as merged as well" - you repeat this statement in different forms several times in this text (e.g. in the frist paragraph of "How it works", and also in Rollup pull requests ). Maybe you can remove one?

Reply:
Removed the first instance, to make the first section smaller and more general.